azure-wildfly/src/test/java/sunstone/azure/armTemplates/archiveDeploy/vmDomain/VmDomainDeploySuiteTests.java

```diff
@@ -13,5 +13,4 @@
 @Suite
 @SelectClasses({AzureDomainVmDeployFirstTest.class, AzureDomainVmUndeployedSecondTest.class})
 public class VmDomainDeploySuiteTests {
-    public static final String vmDeployGroup = "deploytestVM";
```

Comment:
Tests run within this suite must use same resource. Please revert this change and use this group in AzureDomainVmDeployFirstTest and AzureDomainVmUndeployedSecondTest.